Makefile

```diff
@@ -4,13 +4,15 @@ TARGET_DIR=$(CURPATH)/_output
 GOBUILD=go build
 GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 
-DOCKER_OPTS=
-IMAGE_BUILD=docker build
+IMAGE_BUILDER_OPTS=
+IMAGE_BUILDER?=imagebuilder
+IMAGE_BUILD=$(IMAGE_BUILDER)
+IMAGE_TAG?=docker tag
 
 APP_NAME=elasticsearch-operator
 APP_REPO=github.com/openshift/$(APP_NAME)
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
-DOCKER_TAG=github.com/openshift/$(APP_NAME)
+IMAGE_TAG=github.com/openshift/$(APP_NAME)
```

Comment:
I don't think you can have `IMAGE_TAG?=` and `IMAGE_TAG` - maybe rename the first one to `IMAGE_TAG_CMD?=docker tag` ?
Also - `IMAGE_TAG=github.com/openshift/$(APP_NAME)` is not valid - should be `quay.io` or `docker.io` instead of `github.com` - in fact, just omit the domain - just use `openshift/$(APP_NAME)`

Reply:
I updated it to be `quay.io` since this is what we also have in the `cluster-logging-operator` repo's `Makefile`